central/resourcecollection/service/service_impl.go

```diff
@@ -101,10 +101,29 @@ func (s *serviceImpl) getCollection(ctx context.Context, id string) (*v1.GetColl
 	}, nil
 }
 
+// GetCollectionCount returns count of collections matching the query in the request
+func (s *serviceImpl) GetCollectionCount(ctx context.Context, request *v1.GetCollectionCountRequest) (*v1.GetCollectionCountResponse, error) {
+	if !features.ObjectCollections.Enabled() {
+		return nil, errors.New("Resource collections is not enabled")
```

Comment:
The error message it cryptic. how about `errors.Errorf("%s env var is not enabled", {feature_flag}.EnvVar())`